test/scoperesolver_test.js

```diff
@@ -1,10 +1,12 @@
 suite('scoperesolver', () => {
-  let ScopeResolver = require('../src/scoperesolver');
-  let {mergeScopeSets, scopeCompare} = require('taskcluster-lib-scopes');
-  let Monitor            = require('taskcluster-lib-monitor');
-  let assert = require('assert');
-  let _ = require('lodash');
-  let fs = require('fs');
+  const ScopeResolver = require('../src/scoperesolver');
+  const {mergeScopeSets, scopeCompare} = require('taskcluster-lib-scopes');
+  const Monitor            = require('taskcluster-lib-monitor');
```

Comment:
rid us of this horrid space while we're at it!

Reply:
fine :)